app/helpers/application_helper/button/rbac_common_feature_button.rb

```diff
@@ -0,0 +1,11 @@
+class ApplicationHelper::Button::RbacCommonFeatureButton < ApplicationHelper::Button::GenericFeatureButton
+  delegate :rbac_common_feature_for_buttons, :to => :@view_context
+
+  def role_allows_feature?
+    role_allows?(:feature => rbac_common_feature_for_buttons(@feature))
```

Comment:
I was pretty confused at this point, because you check availability of a `@feature` for a role at a lot of buttons, where you don't even set the feature (in toolbar).

After explanation, I've understood, that's why you've changed `feature` loading in previous commit https://github.com/ManageIQ/manageiq-ui-classic/pull/417/commits/af8a4e2382673675ba599292af39b5303c7515de, but still, it seems i bit wrong.

Isn't it better to use `GenericFeatureButton` based class only for the two buttons, that the `rbac_common_feature_for_buttons` method checks?

/cc @PanSpagetka